packages/runtime/datastore/src/dataStoreRuntime.ts

```diff
@@ -402,6 +403,15 @@ export class FluidDataStoreRuntime
 		return context.getChannel();
 	}
 
+	protected validateUserId(id: string) {
+		if (id.includes("/")) {
+			throw new UsageError(`Id cannot contain slashes: ${id}`);
+		}
+		if (id.includes("_")) {
```

Comment:
Need to add a way to opt-out from this new functionality...

Reply:
I checked Loop, AzureClient, Edge - this does not seem to be causing trouble anywhere, so leaving it as is. If we learn it's causing trouble, we will need to reevaluate whole area anyway (i.e. backout and reevaluate).